index.html

```diff
@@ -2,11 +2,155 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
+    <title>Седона</title>
   </head>
+  
   <body>
+    <header class="page-header">
+      <nav class="navigation">
+        <a class="navigation-logo-current" href="index.html">
+          <img class="page-header-logo" alt="Логотип «CITY OF SEDONA»." src="">
+        </a>
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link-current" href="main-page">Главная</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="about.html">О Седоне</a>
+          <li class="navigation-item">
+            <a class="navigation-link" href="hotel.html">Гостиницы</a>
+          </li>
+        </ul>
+        <ul class="navigation-list navigation-user">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#" > 
+              <span class="visually-hidden">
+              Поиск
+            </span>
+          </a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">
+              Лайк
+            </span>
+          </a>
+          </li>
+          <li class="navigation-item">
+            <a class="button-here" href="#">Хочу сюда!</a>
+          </li>
+        </ul>
+       </nav>
+     </header>
+     <main class="main-index">
+      <h1 class="visually-hidden">Sedona</h1>
+      <img class="logo" src="" alt="Логотип Sedona">
```

Comment:
`<img class="logo" src="" alt="Логотип Sedona">` Тут не совсем до конца понятно, что за лого. Скорее всего это относится к блоку с текстом. 

Welcome to georgeous Sedona! Because the Great Canyone sukcs!. 

Весь этот текст нужно будет вырезать как картинку, т.к в макете такой шрифт более нигде не повторяется. А грузить отдельные шрифты только ради одной надписи очень затрано по ресурсам. 

Оставь только блок welcome c img внутри
```
<div class="welcome">
    <img />
</div
```